src/nodeComponents/click/click.js

```diff
@@ -1,14 +1,14 @@
 // dependencies
 import _ from 'lodash';
-import { Vector3 } from 'three';
+import * as THREE from 'three';
 
 // local dependencies
 import store from '../../store/store';
 import C from '../../constants/constants';
 import { simulate } from '../../rootComponents/rootComponents';
 import { getCallbackID, worldToScreenSpace } from '../../util/helpers';
 
-export default (node) =>
+export default node =>
```

Comment:
I know it's a nice feature that ES6 allows to omit parentheses for single arguments, but if we can always keep them, it makes the code more consistent, where we can easily set defaults, add arguments, etc. as needed.